tools/calendar_generation.py

```diff
@@ -0,0 +1,56 @@
+import calendar
+import datetime
+
+monthName = datetime.datetime.now().strftime("%B")
+currentYear = datetime.datetime.now().year
+currentMonth = datetime.datetime.now().month
```

Comment:
These commands are run on import of this script.

In case of teleserver running for e.g. 5 months without restart will these values be updated (I'm sure that it won't be re-imported so these commands will run only once).

Maybe some function to get these values would be better. Or class with all these functions.